src/js/views/patients/widgets/widgets.js

```diff
@@ -0,0 +1,239 @@
+import { each, map, propertyOf, reduce, extend, isFunction } from 'underscore';
+import { View, CollectionView } from 'marionette';
+import dayjs from 'dayjs';
+import Radio from 'backbone.radio';
```

Comment:
A nitpicky question... Is the import order correct here?

Should `Radio` be above `Marionette`?

Reply:
oh yeah probably so